indexer/services/comlink/src/types.ts

```diff
@@ -672,12 +672,26 @@ export interface AffiliateReferralCodeRequest{
   address: string
 }
 
+export interface AffiliateAddressRequest{
+  referralCode: string
+}
+
+export interface AffiliateSnapshotRequest{
+  limit ?: number,
+  offset ?: number,
+  sortByReferredFees ?: boolean
+}
+
+export interface AffiliateTotalVolumeRequest{
+  address: string
+}
+
 export interface AffiliateReferralCodeResponse {
-  referralCode: string | null,
+  referralCode: string | null
```

Comment:
I realize now that every other interface in this file is using semicolons, lets standardize and use a semicolon for every line including the last

Reply:
I think the file is actually pretty mixed. I did a search on ';' and its a bit under 50%. I will add a comma after each element (including single elements and last elements)

Reply:
Actually let me just sync then entire file to one convention.

Reply:
What are your thoughts on adding

`  rules: {
    ...baseConfig.rules,
    'key-spacing': ['error', { beforeColon: false, afterColon: true }],
    // Enforce commas instead of semicolons in interfaces
    '@typescript-eslint/member-delimiter-style': ['error', {
      multiline: {
        delimiter: 'comma',
        requireLast: true,
      },
    }],
  },`

to .eslintrc.js. This would convert all ; to , and also enforce having , for last line